File: 연습장/202308/10_목/13. 나누어 떨어지는 숫자 배열.py
"""
* Title: 나누어 떨어지는 숫자 배열
https://school.programmers.co.kr/learn/courses/30/lessons/12910
"""
def solution(array, divisor):
    answer = []
    for i in array:
        if i % divisor == 0:
            answer.append(i)
    answer.sort()
    if int(len(answer)) == 0:
        return [-1]
    else:
        return answer

File: _프로그래머스 js/13. 나누어 떨어지는 숫자 배열.js
/*
* Title: 나누어 떨어지는 숫자 배열
https://school.programmers.co.kr/learn/courses/30/lessons/12910
*/
const solution = (array, divisor) => {
  let ls = [];

  for (let i = 0; i < array.length; i++) {
    const cur = array[i];

    if (cur % divisor === 0) {
      ls.push(cur);
    }
  }
  if (ls.length === 0) {
    ls.push(-1);
  }

  return ls;
};

// [5, 10]
console.log(solution([5, 9, 7, 10], 5));
